docs/celo-codebase/protocol/bridging/optics-add-new-token.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Optics Add New Token
+description: How to add a new token to Optics v2.
+---
+
+import YouTube from '@components/YouTube';
+import ImageWrapper from '@components/ImageWrapper'
+
+# Add New Token to Optics
+
+How to add a new token to Optics v2.
+
+---
+
+## Getting Started
+
+Optics is a cross-chain communication system designed for use on any EVM chain. The Token Bridge xApp (an application built on top of Optics) facilitates the transfer of tokens between chains. Adding a new token to the Optics Bridge can be done at any time using the [
+optics-bridge-gui-community](https://github.com/celo-org/optics-bridge-gui-community) GitHub repository.
+
+:::info
+
+View [PR#20](https://github.com/celo-org/optics-bridge-gui-community/pull/20/commits/e33b7692b346993c323426418b9b6bba0f5c2096) to view an example of the updates required to add the PACT token.
+
+:::
+
+### Steps to Add a New Token
+
+- **Step 1:** Add Token Identifier
+- **Step 2:** Add Token Symbol
+- **Step 3:** Add Representation Address
+
+## Add Token Identifier
+
+Add the Token Identifier in `src/config/tokens.environment.ts`.
+
+- **Testnet:** [tokens.dev.ts](https://github.com/celo-org/optics-bridge-gui-community/blob/new-deployment/src/config/tokens.dev.ts)
+- **Mainnet:** [tokens.mainnet.ts](https://github.com/celo-org/optics-bridge-gui-community/blob/new-deployment/src/config/tokens.main.ts)
+
+```jsx
+export type TokenIdentifier = {
+  domain: string | number
+  id: string
+}
+
+export type mainTokenName =
+// highlight-next-line
```

Comment:
These `// highlight-next-line` comments make it look kind of weird. I think I prefer not highlighted, wdyt?

<img width="609" alt="Screen Shot 2022-05-03 at 3 28 01 PM" src="https://user-images.githubusercontent.com/18372439/166550855-a6cf1840-b751-47a9-9cc5-9653e40545f1.png">